libs/libarchfpga/src/arch_util.cpp

```diff
@@ -1002,7 +1057,7 @@ e_power_estimation_method power_method_inherited(e_power_estimation_method paren
 void CreateModelLibrary(t_arch* arch) {
     t_model* model_library;
 
-    model_library = new t_model[4];
+    model_library = new t_model[5];
```

Comment:
Define a constant instead of using the magic number 5.

Reply:
Defined a constant for that.